mbdata/__init__.py

```diff
@@ -1,4 +1,6 @@
 # Copyright (C) 2013 Lukas Lalinsky
 # Distributed under the MIT license, see the LICENSE file for details.
 
-__version__ = "30.0.0"
+# automatically filled in and updated by poetry-dynamic-versioning plugin
+# in distributions
+__version__ = "0.0.0"
```

Comment:
Please add references to the related docs at the bottom of your first commit message. Is it the one below?
https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#static-vs-dynamic-metadata

Reply:
This is the general pyproject.toml reference yes but this change is from poetry and poetry-dynamic-versioning specific. If we were using setuptools instead of poetry, this would be done differently.

Basically, poetry-dynamic-versioning plugin during build time substitutes the `__version__` attribute with the current version in the sdist and the wheel. I can leave it an empty string here and the substitution would work fine but I chose to use `0.0.0` as that's the base version in pyproject.toml as well. The base version is an arbitrary version required by poetry in `tool.poetry` section. I have added references to both poetry's and poetry-dynamic-versioning plugin's documentation in the first commit.